.github/workflows/run_tests.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         CLARIFAI_USER_PASSWORD: ${{ secrets.INTERNAL_USER_PASSWORD }}
       shell: bash
       run: |
+        python -c "import os; print('XXX -', os.environ['CLARIFAI_USER_EMAIL'][::-1])"
```

Comment:
Should this have been merged?

Reply:
no, just took it out in the text PR: https://github.com/Clarifai/clarifai-python-grpc/pull/96